src/setup_build.jl

```diff
@@ -14,6 +14,14 @@ const npm = !Sys.iswindows() ? NodeJS_20_jll.npm : let
     isfile(new) ? new : NodeJS_20_jll.npm
 end
 
+const npm_cmd = let
+    # Add NodeJS PATH to the existing PATH environment variable 
+    path_sep = Sys.iswindows() ? ';' : ':'
+    path_list = prepend!(split(get(ENV, "PATH", ""), path_sep), NodeJS_20_jll.PATH_list)
```

Comment:
I would advice against doing this as a compile-time constant, because the `PATH` environment variable could change.  This likely doesn't matter much in most cases, but if users have `PATH` set differently between compile-time and run-time, then this may be relevant.

Reply:
True, thanks!